### DISCUSSION TITLE
Is there "protscnanning" functionality?

### DISCUSSION THREAD

**Discussion by AskAlexSharov** [anacrolix/torrent#708] [Q&A] [answered]
Hi. 
I got some security alert from hosting provider - about "portscanning". Likely it related to my torrent client (base on this lib). 
Can it be related?
Can I somehow configure `anacrolix/torrent` to reduce such risk? (I writing software which will run by other people on various hosting/cloud providers and also on non-server-grade PC). 
Example alert:
```
#              Portscan detected from host   135.181.19.47               #
##########################################################################

time                protocol src_ip src_port          dest_ip dest_port
---------------------------------------------------------------------------
Sat Jan  8 04:17:20 2022 UDP   135.181.19.47 47756 =>      172.17.0.1 55165
Sat Jan  8 04:17:20 2022 UDP   135.181.19.47 47853 =>      172.17.0.1 55165
Sat Jan  8 04:17:52 2022 UDP   135.181.19.47 53290 =>      172.18.0.1 37328
Sat Jan  8 04:17:52 2022 UDP   135.181.19.47 34145 =>      172.18.0.1 37328
Sat Jan  8 04:17:51 2022 UDP   135.181.19.47 35253 =>      172.17.0.1 50235
```

**Comment by anacrolix** [comment, marked as answer]
This is most likely due to the DHT, which would be making a lot of outbound queries to different addresses. Try disabling DHT?

**Reply by AskAlexSharov** [reply]
I do have 
NoDHT = true
DisableTrackers = false
ListenPort = 0

(but I forgot why have "ListenPort = 0" will call `SetListenAddr` with something useful)

**Reply by anacrolix** [reply]
The next thing to try then is to limit the number of concurrent connections, client-wide.

**Reply by AskAlexSharov** [reply]
Thank you, will try
